rotkehlchen/exchanges/binance.py

```diff
@@ -225,8 +230,10 @@ def validate_api_key(self) -> Tuple[bool, str]:
         return True, ''
 
     def api_query(self, method: str, options: Optional[Dict] = None) -> Union[List, Dict]:
-        if not options:
-            options = {}
+        call_options = options.copy() if options else {}
+        # Prevent to pollute next signature
+        if 'signature' in call_options:
```

Comment:
This should actually never be needed right now. I am not completely against defensive programming but if you want to add this at all make it an assertion.

If that ever happens it means it's a programming error, not a user, or input data error.